hibernate-core/src/main/java/org/hibernate/boot/internal/SessionFactoryBuilderImpl.java

```diff
@@ -696,6 +698,8 @@ public SessionFactoryOptionsStateStandardImpl(StandardServiceRegistry serviceReg
 			this.commentsEnabled = ConfigurationHelper.getBoolean( USE_SQL_COMMENTS, configurationSettings );
 
 			this.preferUserTransaction = ConfigurationHelper.getBoolean( PREFER_USER_TRANSACTION, configurationSettings, false  );
+
```

Comment:
I do not like this stuff on SessionFactoryBuilder at all.  SessionFactoryBuilder is for defining settings that get incorporated into the SessionFactory built.